java/cdk-custom-resources/src/test/java/sleeper/cdk/custom/PropertiesWriterLambdaIT.java

```diff
@@ -165,7 +165,7 @@ public void shouldDeleteConfigObjectWhenCalledWithDeleteRequest() throws IOExcep
         AmazonS3 client = createClient();
         String bucketName = UUID.randomUUID().toString();
         client.createBucket(bucketName);
-        client.putObject(bucketName, "config", "foo");
+        client.putObject(bucketName, "instance.properties", "foo");
```

Comment:
Would it be better to use `InstanceProperties.S3_INSTANCE_PROPERTIES_FILE` here in case we change the name in future?

Reply:
Thanks, i'm importing and using that constant in the tests now